oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsUsbDevice.java

```diff
@@ -55,269 +57,256 @@
  */
 public class WindowsUsbDevice extends AbstractUsbDevice {
 
-    private static final Logger LOG = LoggerFactory.getLogger(WindowsUsbDevice.class);
+	private static final Logger LOG = LoggerFactory.getLogger(WindowsUsbDevice.class);
```

Comment:
You may not have switched your formatter to use the OSHI conventions, as all your indents have turned into tabs.

Reply:
Thanks for reminding me! I'll try and fix it right now, and I'll let you know how it goes.

Reply:
All you need to do when you fix it is to push the new commit, and it'll automatically update this request.